corehq/apps/hqadmin/models.py

```diff
@@ -65,6 +68,21 @@ def get_by_pillow_name(cls, pillow_name):
         return store
 
 
+class ESRestorePillowCheckpoints(models.Model):
+    seq = models.TextField()
+    checkpoint_id = models.CharField(max_length=255, db_index=True)
+    date_updated = models.DateField()
+
+    @classmethod
+    def get_pillow_snapshots(cls):
```

Comment:
Since this doesn't return anything I'd rename to `create_pillow_checkpoing_snapshots` or event `snapshot_pillow_checkpoints`